exercises/yacht/examples/success-standard/src/Yacht.hs

```diff
@@ -0,0 +1,69 @@
+module Yacht (yacht, Category(..)) where
+
+import Data.List
+
+data Category = Ones
+              | Twos
+              | Threes
+              | Fours
+              | Fives
+              | Sixes
+              | FullHouse
+              | FourOfAKind
+              | LittleStraight
+              | BigStraight
+              | Choice
+              | Yacht
+
+yacht :: Category -> [Int] -> Int
+yacht _ []   = error "You lose!"
+yacht Ones   dice = simpleScore 1 dice
+yacht Twos   dice = simpleScore 2 dice
+yacht Threes dice = simpleScore 3 dice
+yacht Fours  dice = simpleScore 4 dice
+yacht Fives  dice = simpleScore 5 dice
+yacht Sixes  dice = simpleScore 6 dice
+
+yacht FullHouse dice
+  | s == 2 && length g == 2 && (length . head') gs == 3 = sum dice
+  | otherwise = 0
+  where
+    s = (length . gsortBySize) dice
+    (g:gs) = gsortBySize dice
+
+yacht FourOfAKind dice
+  | length lg == 4 = sum lg
+  | length lg == 5 = 4 *  (head dice)
```

Comment:
Unnecessary double space and parenthesis.
```suggestion
  | length lg == 5 = 4 * head dice
```

Reply:
Cool. Extra space was an accident.  I tried `$ head` and it failed (that should have been an indication that I didn't need to force it's evaluation first 😁)